### DISCUSSION TITLE
EIP3091 standard

### DISCUSSION THREAD

**Discussion by tomatoskittles** [blockscout/blockscout#4755] [Q&A] [answered]
Hey, I was just wondering if anyone could tell me if Blockscout is EIP3091 compliant, or not.

Thanks in advance!

**Comment by vbaranov** [comment, marked as answer]
Hi, EIP-3091 compliance has been added in 
> #4655, #4676 - EIP-3091 support
 https://github.com/blockscout/blockscout/blob/master/CHANGELOG.md

It is available in master branch and will be included in the upcoming release.

**Reply by tomatoskittles** [reply]
Thank you for the information!
